app/src/main/kotlin/cz/covid19cz/erouska/ui/dashboard/DashboardFragment.kt

```diff
@@ -50,6 +50,7 @@ class DashboardFragment : BaseFragment<FragmentPermissionssDisabledBinding, Dash
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        activity?.setTitle(R.string.app_name)
```

Comment:
Jakej byl tady problem? Nemel by ten title nastavovat system podle toho kam se navigujes?

Reply:
Jo ale LoginFragment to nastavuje mimo navigaci (pro každý stav) a to tuhle premisu rozbíjí